modules/xfem/include/base/XFEM.h

```diff
@@ -172,7 +172,7 @@ class XFEM : public XFEMInterface
    */
   std::vector<Real> & getXFEMCutData();
   void setXFEMCutData(std::vector<Real> & cut_data);
-  std::string & getXFEMCutType();
+  // std::string & getXFEMCutType();
```

Comment:
What happened here?

Reply:
OK - normally we'd just delete it then :)

Reply:
I just talked to @bwspenc and ended up removing a few more member declarations of methods that were factored away a while ago. The non-const string arguments of at least one of those puts this in the issue scope IMO.